app/Api/V1/Requests/Models/Rule/ValidateExpressionRequest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * ValidateExpressionRequest.php
+ * Copyright (c) 2024 Michael Thomas
+ *
+ * This file is part of Firefly III (https://github.com/firefly-iii).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+declare(strict_types=1);
+
+namespace FireflyIII\Api\V1\Requests\Models\Rule;
+
+use FireflyIII\Support\Request\ChecksLogin;
+use FireflyIII\Support\Request\ConvertsDataTypes;
+use Illuminate\Foundation\Http\FormRequest;
+
+/**
+ * Class ValidateExpressionRequest
+ */
+class ValidateExpressionRequest extends FormRequest
```

Comment:
Add a `rules()` function that returns something like `['expression' => 'required|ruleActionExpression']`

Reply:
See also the comment about the class/method for validation.

Reply:
Took a stab at doing it this way, still not sure how I feel about it. Please let me know if there's a better way to do this.